cypress/fixtures/no-freqId-version-phy-device.json

```diff
@@ -0,0 +1,229 @@
+[
+  {
+    "ids": {
+      "device_id": "no-fallback-test-device",
+      "dev_eui": "008BFC3FD4445B23",
+      "join_eui": "70B3D57ED0042F43"
+    },
+    "created_at": "0001-01-01T00:00:00Z",
+    "updated_at": "0001-01-01T00:00:00Z",
+    "name": "migration-test-device",
+    "supports_join": true,
+    "root_keys": {
+      "app_key": {
+        "key": "EFE35B55F8B394A68567F8A36F5AC44A"
+      }
+    },
+    "mac_settings": {
+      "supports_32_bit_f_cnt": true,
+      "status_time_periodicity": "0s",
+      "status_count_periodicity": 0
+    },
+    "mac_state": {
```

Comment:
The mac state prop here can be removed as well.

Reply:
I removed it and when importing an `internal server error` was thrown. So I put it back